### ISSUE TITLE
[Bug] JSON numerics longer than 16 digits are transformed to a different value

### ISSUE THREAD

**Issue by RyanObray** [usebruno/bruno#1689]
## Summary ##
When processing JSON bodies, Bruno transforms numeric values longer than 16 digits in length, resulting in incorrect data passing to the target endpoint.

## Reproduction ##
This is quite easy to reproduce in two ways.

**Method 1**
Create a JSON POST request with the following body:
``` JSON
[
  {
    "id": 1234567890123456,
    "index": 1
  },
  {
    "id": 12345678901234567,
    "index": 2
  },
  {
    "id": 123456789012345678,
    "index": 3
  },
  {
    "id": 1234567890123456789,
    "index": 4
  },
  {
    "id": 12345678901234567890,
    "index": 5
  }
]
````

Notice that item 1 (index 1) has a 16 digit ID and that all subsequent items have one more digit than the previous. 

Click "Timeline" in Bruno and then execute the request. After executing the request, review the payload "data" in the "Timeline" pane and note that Bruno modified all digits after the 16th place in the IDs. This results in incorrect IDs being passed to the endpoint.

Here's a screenshot of the behavior:

![image](https://github.com/usebruno/bruno/assets/60630345/92725075-ec79-4209-a3bb-2540da81f822)

**Method 2**
With the same JSON listed in method 1 still in the "Body", simply hit the "Prettify" button. This will update the request with the same morphed values seen above.

Here are before and after screenshots, illustrating the bug:

Before hitting "Prettify"
![image](https://github.com/usebruno/bruno/assets/60630345/cfe3b103-d568-452a-bfa7-265b109dbca7)

After hitting "Prettify"
![image](https://github.com/usebruno/bruno/assets/60630345/bcb04265-dd78-4931-95cd-92b9e6478e51)

I realize that this is most likely caused by limitations of  JSON.parse() and that it rounds large numbers. I also realize that it can technically be avoided by wrapping the numbers in quotes and treating them like strings, but doing that feels dirty and puts a burden on users to transform JSON they might be pasting in from their system etc. I have confirmed that this is not a problem in Postman or Insomnia so it may be worth considering.

**Comment by laufermi** [comment]
I encountered the same issue where I retrieve a SnowflakeId from the database (BIGINT), read it into Long in Java, and request api via Bruno

mysql:
<img width="244" alt="image" src="https://github.com/usebruno/bruno/assets/74701117/35315369-4446-4ed4-ac33-6b7555c1b5cc">


curl:
<img width="454" alt="image" src="https://github.com/usebruno/bruno/assets/74701117/d397b25c-614b-43e1-8561-9fbf8c82ab6e">

bruno:
<img width="622" alt="image" src="https://github.com/usebruno/bruno/assets/74701117/da2570f9-e291-4cb8-8a6b-e123357ea75f">

**Comment by duc3d-hus** [comment]
@laufermi me too :(

**Comment by sanjai0py** [comment]
Hi @RyanObray, the issue you reported has been fixed in #1710 and will be available to the public in next release. Please close the issue. Thank you!

**Comment by RyanObray** [comment]
Closing due to #1710 fix.
